From 0bb4ace0877e203d7cca2bc28c484ef5ee0bbabe Mon Sep 17 00:00:00 2001
From: Shuguang Gong <shuguang.gong@intel.com>
Date: Thu, 19 Mar 2015 16:45:31 +0800
Subject: [PATCH 318/323] ov2685: remove 720p setting

Current 720p resolution is cropped by sensor from full size.
This causes some FoV degree issue for sensor limitation.
Removing the 720p setting to make it to be cropped by ISP
to solve the FoV degree issue.

Change-Id: I04f82684eeade5323b305977ca75fc646b237d17
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-25392
Signed-off-by: Shuguang Gong <shuguang.gong@intel.com>
Signed-off-by: Tao Wei <wei.tao@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/47058
Build: IMIN-legacy (SC)
Approver: Jing Tao <jing.tao@intel.com>
---
 .../external_drivers/camera/drivers/media/i2c/ov2685.h   | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h
index 5a9f33a..65f21c8 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov2685.h
@@ -258,22 +258,6 @@ enum {
 
 static struct ov2685_res_struct ov2685_res[] = {
 	{
-	.desc	= "720P",
-	.res	= OV2685_RES_720P,
-	.width	= 1280,
-	.height	= 720,
-	.pixels_per_line = 1446,
-	.lines_per_frame = 760,
-	.fps	= 30,
-	.pix_clk = 33,
-	.used	= 0,
-	.regs	= NULL,
-	.skip_frames = 1,
-	.lanes = 1,
-	.bin_mode = 0,
-	.bin_factor_x = 0,
-	},
-	{
 	.desc	= "2M",
 	.res	= OV2685_RES_2M,
 	.width	= 1600,
-- 
1.9.1

